tests/PHPStan/Analyser/data/conditional-types-constant.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace ConditionalTypesConstant;
+
+use function PHPStan\Testing\assertType;
+
+abstract class Test
+{
+	/**
+	 * @return (PHP_MAJOR_VERSION is 8 ? true : false)
```

Comment:
Obviously this breaks on PHP < 8 :boom:

Reply:
you should run this test only on php8, or add an php7 alternative test-case with a different expectation.